Sources/App/Controllers/AppUserController.swift

```diff
@@ -0,0 +1,78 @@
+import Vapor
+
+struct AppUserController {
+    func logIn(request: Request) -> EventLoopFuture<DataWrapper<AppUserLoginResponse>> {
+        AppUserLoginRequest
+            .logIn(
+                on: request,
+                errorOnWrongPassword: AppError.incorrectCredentials,
+                currentDate: request.date.current()
+            )
+            .flatMapThrowing { try $0.map(AppUserMeResponse.init) }
+            .map(DataWrapper.init)
+    }
+
+    func refreshToken(request: Request) throws -> DataWrapper<RefreshTokenResponse> {
+        try .init(data: .init(AppUserRefreshKeychainConfig.makeToken(on: request)))
+    }
+
+
+    func create(request: Request) throws -> EventLoopFuture<Response> {
+        AppUserCreateRequest
+            .create(on: request)
+            .flatMap(request.repositories.appUser.save)
+            .flatMapThrowing(AppUserCreateResponse.init)
+            .map(DataWrapper.init)
+            .encodeResponse(status: .created, for: request)
+    }
+
+    func list(request: Request) throws -> EventLoopFuture<NodesPage<AppUserResponse>> {
+        let searchterm = try request.query.get(String?.self, at: "searchterm")
```

Comment:
Could you use the same extension as we have in the BAT mood project to get the search term? I think that sets a good pattern.